Source/Core/DolphinWX/Input/InputConfigDiag.h

```diff
@@ -105,13 +105,13 @@ class ControlDialog : public wxDialog
 {
 public:
   ControlDialog(InputConfigDialog* const parent, InputConfig& config,
-                ControllerInterface::ControlReference* const ref);
+                ControlReference* const ref);
 
   bool Validate() override;
 
   int GetRangeSliderValue() const;
 
-  ControllerInterface::ControlReference* const control_reference;
+  ControlReference* const control_reference;
```

Comment:
It probably shouldn't, but I haven't delved into InputConfigDiag at all so I don't know what depends on it. I'd rather leave this section of the code alone for now, if that's okay.